### PR TITLE
added tox as requirement to readme - added gnss, imu, and user to msg.js

### PR DESCRIPTION
**Pull request by jkretzmer**
added pip install tox to readme
added gnss, imu, and user to msg.js

### CODE REVIEW

File: javascript/README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ to parse individual binary messages.packets.
 
 ### Requirements
 ```
+pip install tox
```
Comment (mookerji) [inline javascript/README.md]: @jkretzmer yo, what's tox being used for here? Isn't that only used in the Python bindings?

File: javascript/sbp/msg.js [hunk 2]
```diff
@@ -93,13 +93,16 @@ var sbpImports = {
   ext_events: require('./ext_events.js'),
   file_io: require('./file_io.js'),
   flash: require('./flash.js'),
+  gnss: require('./gnss.js'),
```
Comment (mookerji) [inline javascript/sbp/msg.js]: Great, nice catch!

**Comment by mookerji** [comment]
@jkretzmer see question about tox.

File: javascript/README.md (hunk 1 above)
Comment (jkretzmer) [reply javascript/README.md]: from the top level Makefile

the javascript target is:
`javascript: deps-javascript gen-javascript test-javascript`

gen-javascript is:
`gen-javascript:
	$(call announce-begin,"Generating JavaScript bindings")
	cd $(SWIFTNAV_ROOT)/generator; \
	$(SBP_GEN_BIN) -i $(SBP_SPEC_DIR) \
		       -o $(SWIFTNAV_ROOT)/javascript/sbp/ \
                       -r $(SBP_MAJOR_VERSION).$(SBP_MINOR_VERSION) \
		       --javascript
	cd $(SWIFTNAV_ROOT)
	$(call announce-begin,"Bumping NPM version")
	@- npm version "v$(SBP_MAJOR_VERSION).$(SBP_MINOR_VERSION).$(SBP_PATCH_VERSION)" --no-git-tag-version >/dev/null 2>&1
	$(call announce-end,"Finished generating JavaScript bindings. Please check $(SWIFTNAV_ROOT)/javascript/sbp")`

The SBP_GEN_BIN is tox. `make javascript` fails without it being installed.

**Comment by jkretzmer** [comment]
@mookerji - see above explanation re: tox - are we good to merge?

**Comment by mfine** [comment]
:shipit:

File: javascript/README.md (hunk 1 above)
Comment (mookerji) [reply javascript/README.md]: ooof

**Comment by mookerji** [comment]
thanks @jkretzmer !

**Pull request merged**